src/pages/billing/Project.js

```diff
@@ -308,7 +319,56 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
               isClearable: false,
               options: _.map(({ displayName, accountName }) => ({ label: displayName, value: accountName }), billingAccounts),
               onChange: ({ value: newAccountName }) => setSelectedBilling(newAccountName)
+            }),
+            div({ style: { marginTop: '1rem' } },
+              ['Note: Changing the billing account for this billing project will clear the workflow spend report configuration.'])
```

Comment:
Generally the style here for linebreaks would be to keep the braces on the same line as the div, so that there's a closing line for the div at the same indentation as the opening.:
```js
div({ props }, [
  'content'
])
```